policies/intel-aws-db-instance-deny-unapproved-instance-types/intel-aws-db-instance-deny-unapproved-instance-types.sentinel

```diff
@@ -0,0 +1,127 @@
+import "tfplan/v2" as tfplan
+
+param valid_actions default [
+	["no-op"],
+	["create"],
+	["update"],
+]
+
+format = {
+	"reset": "\033[0m",
+	"info":  "\033[32m",
+	"warn":  "\033[33m",
+	"error": "\033[31m",
+}
+
+summary = func(input, level) {
+	result = false
+	if input.violations is empty {
+		return true
+	} else {
+		log(input, format[level])
+	}
+	return result
+}
+
+log = func(input, output) {
+	header(input, output)
+	violations(input, output)
+	return null
+}
+
+header = func(input, output) {
+	print(
+		output + "\t========================================================================\n",
+		output + "\t                    _       _       _ \n",
+		output + "\t                   (_)     | |     | |\n",
+		output + "\t                    _ _ __ | |_ ___| |\n",
+		output + "\t                   | | '_ \\| __/ _ \\ |\n",
+		output + "\t                   | | | | | ||  __/ |\n",
+		output + "\t                   |_|_| |_|\\__\\___|_|\n",
+		output + "\t\n",
+		output + "\t========================================================================\n",
+		output + "\tName        :" + output + " " + "intel-aws-db-instance-deny-unapproved-instance-types.sentinel\n",
+		output + "\tCategory    :" + output + " " + "Infrastructure (IaaS)\n",
+		output + "\tProvider    :" + output + " " + "hashicorp/aws\n",
+		output + "\tResource    :" + output + " " + "aws_db_instance\n",
+		output + "\tParameter   :" + output + " " + "instance_class\n",
+		output + "\tCheck       :" + output + " " + "instance_class does not contain\n",
+		output + "\t\n",
+		output + "\t              db.m6i.large, db.m6i.xlarge\n",
+		output + "\t              db.m6i.2xlarge, db.m6i.4xlarge\n",
+		output + "\t              db.m6i.8xlarge, db.m6i.12xlarge\n",
+		output + "\t              db.m6i.16xlarge, db.m6i.24xlarge\n",
+		output + "\t              db.m6i.32xlarge, db.r6i.large\n",
+		output + "\t              db.r6i.xlarge, db.r6i.2xlarge\n",
+		output + "\t              db.r6i.4xlarge, db.r6i.8xlarge\n",
+		output + "\t              db.r6i.12xlarge, db.r6i.16xlarge\n",
+		output + "\t              db.r6i.24xlarge, db.r6i.32xlarge\n",
+	)
```

Comment:
We should look at moving all of the values we are evaluating to a markdown file in a folder for the policy. Shift the output values to providing a link to what we are evaluating so that way we aren't having to maintain the list in print statements and also apply formatting to each line item.

Reply:
Agreed - targeting for next release